app/src/main/java/com/celzero/bravedns/util/Utilities.kt

```diff
@@ -482,5 +489,79 @@ class Utilities {
         fun isAtleastR(): Boolean {
             return Build.VERSION.SDK_INT >= Build.VERSION_CODES.R
         }
+
+        fun isAtleastQ(): Boolean {
+            return Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q
+        }
+
+        fun isFdroidBuild(): Boolean {
+            return BuildConfig.FLAVOR == FLAVOR_FDROID
+        }
+
+        fun isPlayStoreBuild(): Boolean {
+            return BuildConfig.FLAVOR == FLAVOR_PLAY
+        }
+
+        fun getBugReportFilePath(context: Context): String {
+            val filePath = context.filesDir.canonicalPath + File.separator + Constants.BUG_REPORT_FILE
+            val file = File(filePath)
+            if (file.exists()) {
+                // Only write if the file is less than 10mb
+                if (isFileLessThan10MB(file)) {
```

Comment:
why not create a directory of files and zip them up instead of one big file?

https://archive.is/k0VIa 

In case of `zip`, `10MB` may be pretty sufficient for a _lot_ of bugreports.

Reply:
Added as TODO.

Reply:
Lets address it with this commit. Zip the files when they are stored or emailed. Unzip them to read.

Reply:
zipped when stored, now.